pkg/config/config.go

```diff
@@ -99,24 +99,60 @@ func (c *Config) FindSiteByHostName(hostname string) (*Site, error) {
 	return nil, fmt.Errorf("unable to find site with hostname %s", hostname)
 }
 
-// ListOfSitesByDirectory takes the users home directory and the current
-// working directory and returns a list of sites that are "context-aware".
+// ListOfSitesByDirectory takes the user’s home directory and the current
+// working directory and returns a list of sites within that context.
 func (c *Config) ListOfSitesByDirectory(home, wd string) []Site {
 	var found []Site
+
+	// get any sites whose container paths are within the working directory
 	for _, s := range c.Sites {
-		p, _ := s.GetAbsPath(home)
+		p, _ := s.GetAbsContainerPath(home)
```

Comment:
Not sure if the container path should come into play here? We should discuss and walk through when we can

Reply:
Ended up having to keep it per our discussion.